doc/documentation_data_format.md

```diff
@@ -109,36 +109,47 @@ Any parameters named `noiseParameter${1..n}` *must* be overwritten in the
 
 ## Condition table
 
-The condition table specifies parameters or *constant* species for specific
+The condition table specifies parameters, initial values of species and compartments for specific
 simulation conditions (generally corresponding to different experimental
 conditions).
 
 This is specified as tab-separated value file with condition-specific
-species/parameters in the following way:
+species/parameters/compartments in the following way:
 
-| conditionId | [conditionName] | parameterOrStateId1 | ... | parameterOrStateId${n} |
+| conditionId | [conditionName] | parameterOrStateOrCompartmentId1 | ... | parameterOrStateOrCompartmentId${n} |
 |---|---|---|---|---|
-| conditionId1 | conditionName1 | NUMERIC&#124;parameterId | ...| ...
+| conditionId1 | conditionName1 | NUMERIC&#124;parameterId&#124;stateId&#124;compartmentId | ...| ...
 | conditionId2 | ... | ... | ...| ...
 |... | ... | ... | ... |...| ...
 
 Row names are condition names as referenced in the measurement table below.
-Column names are global parameter IDs or IDs of constant species as given in
-the SBML model. These parameters will override any parameter values specified
-in the model. `parameterOrStateId`s and `conditionId`s must be unique.
+Column names are global parameter IDs, IDs of species or compartments as given
+in the SBML model. These parameters will override any parameter values
+specified in the model. `parameterOrStateId`s and `conditionId`s must be
+unique.
+
+If a species ID is provided, it is interpreted as the initial
+concentration/amount of that species and will override the initial
+concentration/amount given in the SBML model or given by a preequilibration
+condition. If `NaN` is provided for a condition, the result of the
+preequilibration (or initial concentration/amount from the SBML model, if no
+preequilibration is defined) is used.
```

Comment:
If you specify *all* initial conditions here, and if for the simulationCondition all of them are non-NaN, then you could as well skip preequlibration. This is only useful (and sometimes required, since you generally can't just preequilibrate a subset of the model) if you do not add/overwrite all initial conditions here.